OpenKeychain/src/main/java/org/sufficientlysecure/keychain/service/KeychainIntentService.java

```diff
@@ -61,10 +61,7 @@
 import org.sufficientlysecure.keychain.provider.ProviderHelper;
 import org.sufficientlysecure.keychain.service.input.CryptoInputParcel;
 import org.sufficientlysecure.keychain.service.ServiceProgressHandler.MessageStatus;
-import org.sufficientlysecure.keychain.util.FileHelper;
-import org.sufficientlysecure.keychain.util.InputData;
-import org.sufficientlysecure.keychain.util.Log;
-import org.sufficientlysecure.keychain.util.ParcelableFileCache;
+import org.sufficientlysecure.keychain.util.*;
```

Comment:
Sorry! Must have crept in when I reformatted code. I'll remove it before the final push.